src/main/resources/META-INF/resources/index.html

```diff
@@ -96,16 +83,19 @@
         }
 
     </style>
+    <link rel="stylesheet" href="navbar.css" />
```

Comment:
@redcrafter502 Du hast die Navbar-Styles in eine eigene Datei verschoben. Das ist gut. Aber wäre es dann nicht sinnvoll, alle CSS-Styles aus dieser Datei in eine eigene zu verschieben?

Reply:
@methom Das wäre aber was für ein anderen pull request und ein anderes Issue.

Reply:
👍